designcompose/src/main/java/com/android/designcompose/Utils.kt

```diff
@@ -54,6 +54,7 @@ import com.android.designcompose.serdegen.BlendMode
 import com.android.designcompose.serdegen.Dimension
 import com.android.designcompose.serdegen.Display
 import com.android.designcompose.serdegen.Easing
+import com.android.designcompose.serdegen.FigmaColor
```

Comment:
Oops, `FigmaColor` is a type used in `figma_schema.rs`. It represents the structure of a color when parsed from the Figma response. The color we use in `toolkit_schema.rs` should be a different `Color` struct and I'll fix that so that we don't output it into code that Kotlin uses.